hpcrs/Dockerfile

```diff
@@ -173,6 +86,26 @@ RUN mkdir -p /tmp/intel_shm/4k/ /tmp/intel_shm/2m /tmp/intel_shm/1g \
     && touch /etc/security/limits.conf \
     && echo "root hard memlock unlimited" >> /etc/security/limits.conf \
     && echo "root soft memlock unlimited" >> /etc/security/limits.conf \
-    && rm -rf /workspace/components
+    && cp components/licenses/third-party-programs_v3.txt   /workspace/ \
```

Comment:
nit: This chunk of code might be shorten in a loop deleting n amount of extentions in an array.

But again, this is enterily up to the developer architectural decition of course.
Overall this looks good to me.